pandas/tests/series/test_arithmetic.py

```diff
@@ -679,6 +680,38 @@ def test_series_add_aware_naive_raises(self):
         with pytest.raises(Exception, match=msg):
             ser_utc + ser
 
+    def test_series_add_daytime_offset(self):
+        # GH#19211
+        # Ignore PerformanceWarning for this test case
+        warnings.simplefilter(action="ignore", category=pd.errors.PerformanceWarning)
+        ser_daytime = pd.Series(
+            [pd.Timestamp("2000-01-01"), pd.Timestamp("2000-02-01")]
+        )
+        ser_offset = pd.Series(
+            [pd.offsets.DateOffset(years=1), pd.offsets.DateOffset(months=2)]
+        )
+
+        result = ser_daytime + ser_offset
```

Comment:
The original issue has this raising when the offset is on the left so we would want to test that. You could try parametrizing this such that we test both timestamp + offset and offset + timestamp, with both yielding the same answer.